doc/source/getting_started/dsintro.rst

```diff
@@ -397,6 +397,28 @@ The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+From a list of dataclasses
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.0.0
+dataclasses as introduced in `PEP557 <https://www.python.org/dev/peps/pep-0557>`,
```

Comment:
Can you leave a blank line between the `.. versionadded:: ` and this one? (sphinx is rather picky about that ..)